preppal-be/models/recipe.ts

```diff
@@ -50,6 +50,12 @@ const RecipeSchema = new Schema({
     },
     modifiedDate: {
         type: Date
+    },
+    tags: {
+        type: Array
+    },
+    visibility: {
+        type: String
```

Comment:
Why String and not a simple boolean flag like true for public, false for private?

Reply:
fixed